cardano_node_tests/tests/plutus_common.py

```diff
@@ -41,6 +41,7 @@ class PlutusOp(NamedTuple):
     datum_cbor_file: Optional[Path] = None
     redeemer_file: Optional[Path] = None
     redeemer_cbor_file: Optional[Path] = None
+    redeemer_value: Optional[str] = ""
```

Comment:
Use `None` default value, since you are using `Optional` anyway, and it will be consistent with the rest of the items.

Reply:
in fact I used an empty string to keep the same default value as the one present in clusterlib  ScriptTxIn(NamedTuple), but I will change it, no problem :)